libs/external-router/driver/in-memory/src/config.ts

```diff
@@ -31,4 +31,19 @@ export const {
    * Provider function for {@link DAFF_EXTERNAL_ROUTER_DRIVER_IN_MEMORY_CONFIG}.
    */
   provider: provideDaffExternalRouterDriverInMemoryConfig,
-} = createConfigInjectionToken<DaffExternalRouterDriverInMemoryConfig>(defaultConfig, 'DAFF_EXTERNAL_ROUTER_DRIVER_IN_MEMORY_CONFIG');
+} = createConfigInjectionToken<DaffExternalRouterDriverInMemoryConfig>(undefined, 'DAFF_EXTERNAL_ROUTER_DRIVER_IN_MEMORY_CONFIG', {
```

Comment:
Should we improve `createConfigInjectionToken` to allow default config to be a function ran in the injection context?

Reply:
Well the config is typed such that the only required fields are those that are not present in the default. This doesn't work if the default is provided through the factory.